ddspipe_core/src/cpp/communication/dds/Track.cpp

```diff
@@ -238,7 +239,7 @@ void Track::transmit_() noexcept
                 continue;
             }
         }
-        else if (!ret)
+        else if (ret != utils::ReturnCode::OK)
         {
             // Error reading data
             logWarning(DDSPIPE_TRACK, "Error taking data in Track " << topic_->serialize() << ". Error code " << ret
```

Comment:
logWarning, logError and logInfo macros are deprecated. Please change them for EPROSIMA_LOG_WARNING, EPROSIMA_LOG_ERROR and EPROSIMA_LOG_INFO respectively.
```suggestion
            EPROSIMA_LOG_WARNING(DDSPIPE_TRACK, "Error taking data in Track " << topic_->serialize() << ". Error code " << ret
```